pkg/quarterdeck/auth.go

```diff
@@ -589,12 +590,13 @@ func (s *Server) Refresh(c *gin.Context) {
 	user, err := models.GetUser(c, claims.Subject, orgID)
 	if err != nil {
 		if errors.Is(err, models.ErrUserOrganization) {
-			c.JSON(http.StatusForbidden, api.ErrorResponse("invalid credentials"))
+			c.JSON(http.StatusForbidden, api.ErrorResponse(responses.ErrLogBackIn))
 			return
 		}
 
 		sentry.Warn(c).Err(err).Msg("could not retrieve user from database")
-		c.JSON(http.StatusForbidden, api.ErrorResponse("could not retrieve user from claims"))
+		// Should this be an http.StatusInternalServerError instead? :point-down:
+		c.JSON(http.StatusForbidden, api.ErrorResponse(responses.ErrLogBackIn))
```

Comment:
I'm assuming `http.StatusForbidden` is currently used here because if a user can't be retrieved from the database, then they shouldn't have access. If the user is told to log in again, then I think that `http.StatusInternalServerError` works better because we would not want to inform a user that they should try logging in again while also returning a forbidden error.